compiler/src/dotty/tools/dotc/reporting/trace.scala

```diff
@@ -92,4 +100,11 @@ object trace {
         throw ex
     }
   }
+}
+
+object trace extends TraceSyntax {
```

Comment:
Are you using this?

Reply:
This replaces the previous `trace` object - now wherever there are calls like `trace("isSubtype") { ... }` it's possible to replace `trace` with `trace.force` to have only that particular trace printed.

Reply:
I see. Then that's maybe a could place to put two lines of scaladoc...